### PR TITLE
Optionally use Gregorian calendar parser when global locale is 'fa'

### PR DESCRIPTION
**Pull request by pouriaMaleki**
As we discussed in #49, I have created an option proposal, to use Gregorian calendar parser like this:

```javascript
    moment.locale('fa', { useGregorianParser: true });  
    moment('2018-04-04').format();// it would be in jalali system  
    moment('2019-01-17T08:19:19.975Z').format();// it would be in jalali system  
```

I tried to make minimum possible change in code. Tests and docs are also added.

### CODE REVIEW

**Comment by fingerpich** [comment]
Awesome work!Very impressive :)
There is a problem with travis, because of different timezones, please just don't check `HH:mm:ss`.

**Comment by pouriaMaleki** [comment]
Thank you @fingerpich for quick action and for awesome library. :-)

I have fixed the tests and bumped the npm package version in `package.json`.

**Comment by fingerpich** [comment]
Its not awesome I think its just usable :)
I changed the version because you have added a feature and now its available in [npm](https://npmjs.com/package/jalali-moment).
Thanks a lot.

File: jalali-moment.js [hunk 1]
```diff
@@ -989,8 +989,13 @@ jMoment.unBindCalendarSystemAndLocale = function () {
     moment.changeCalendarSystemByItsLocale = false;
 };
 
-jMoment.useJalaliSystemPrimarily = function () {
+jMoment.useJalaliSystemPrimarily = function (options) {
     moment.justUseJalali = true;
+    useGregorianParser = false;
```
Comment (miladmeidanshahi) [inline jalali-moment.js]: useGregorianParser is not defined, use var

**Pull request merged**